conf/README.md

```diff
@@ -92,9 +98,9 @@ First set up the webwork2 Mojolicious app:
 
 It is not advisable to run the Mojolicious hypnotoad server as the root user or as any user that can directly login to
 the server.  Instead use a user that does not have a login account.  On Ubuntu systems you can use the `www-data` user
-that is already available.  If a user is needed you can create the user `webwork`, for example, with `sudo useradd -M
-webwork`.  Make sure that the user has read access to the ssl certificates given in the configuration above if using
-certificates.
+that is already available.  If a user is needed you can create the user `webwork`, for example, with
+`sudo useradd -M webwork`.  Make sure that the user has read access to the ssl certificates given in the configuration
+above if using certificates.
```

Comment:
This change isn't necessary.  Markdown displays this correctly the way it was.  As to if `sudo useradd -M webwork` wraps or not is entirely dependent on the viewing window size, and it makes no difference if it is wrapped here or not.

Reply:
Ok.  The built-in markdown browser in my text editor wasn't displaying that correct.  That's why I put in on a separate line.

Reply:
You can leave it as you have it.  It is kind of nice having it not wrapped in the the file itself.